src/features/views/store.ts

```diff
@@ -180,6 +178,46 @@ const viewsSlice = createSlice({
         }
       }
     },
+    columnOrderUpdated: (state, action: PayloadAction<[number, number[]]>) => {
+      const [viewId, columnOrder] = action.payload;
+      // Re-arrange columns
+      const colList = state.columnsByViewId[viewId];
+      if (colList) {
+        const newColListItems = columnOrder.map((colId) => {
+          const col = colList.items.find((col) => col.id == colId);
+          if (col) {
+            return col;
+          } else {
+            throw new Error('Could not find column!');
```

Comment:
This should never happen, but if it does, it will now crash the app. Instead of throwing an error and crashing, wouldn't it be better to just mark the column list as stale and force it to reload?